Mathlib/MeasureTheory/Integral/CircleIntegral.lean

```diff
@@ -177,7 +177,9 @@ theorem circleIntegrable_const (a : E) (c : ℂ) (R : ℝ) : CircleIntegrable (f
 
 namespace CircleIntegrable
 
-variable {f g : ℂ → E} {c : ℂ} {R : ℝ}
+variable
```

Comment:
If this fits all on one line, I'd fit it.

Reply:
Done.